src/phoenix/server/api/helpers/playground_spans.py

```diff
@@ -64,10 +59,9 @@ class streaming_llm_span:
     def __init__(
```

Comment:
I'm not 100% convinced that the context manager semantics here feel right—it's not super clear what context this needs to manage in order for it to work and what interfaces it uses, I'm happy to brainstorm later

Reply:
Let's sync

Reply:
Based on feedback offline, improved context manager interface in [44b0050](https://github.com/Arize-ai/phoenix/pull/5237/commits/44b0050dea34a4422f2295a3096d5e3aa2d59e2c)